frontends/web/src/components/bitboxbase/unlockbitboxbase.tsx

```diff
@@ -98,7 +94,7 @@ class UnlockBitBoxBase extends Component<Props, State> {
                                                 id="password"
                                                 type="password"
                                                 label="Password"
-                                                placeholder="Enter your BitBoxBase password to unlock"
+                                                placeholder={t('bitboxBaseWizard.pairing.unpaired')}
                                                 onValidPassword={this.handleFormChange}
                                                 value={password} />
```

Comment:
is it a problem if this is `undefined`? Maybe it is clearer to make it `: string` only and init to `''` after all. Sry I didn't see it before.

Reply:
PasswordSingleInput has its own state and defaults to empty string :) it is controlled and takes the value from the input event target. This prop doesn't do anything, you could just remove it here.